### PR TITLE
Fix foundry.security -> foundry.admin break

### PR DESCRIPTION
**Pull request by bryantpark04**
Fix SDK generation and regen SDK

### CODE REVIEW

File: packages/platform-sdk-generator/src/model/Model.ts [hunk 1]
```diff
@@ -175,7 +175,10 @@ export class Model {
   }
 
   #addComponent(c: ir.Component) {
-    const ns = this.#namespaces.get(c.namespace ?? "");
+    const nsName = (c.namespace === undefined || c.namespace === "Core")
```
Comment (ssanjay1) [inline packages/platform-sdk-generator/src/model/Model.ts:178]: lets do `==null`, read https://pages.github.palantir.build/palantir/documentation/#/gotham/web-standards/typescript/explicitly-check-null

Comment (bryantpark04) [inline packages/platform-sdk-generator/src/model/Model.ts:178]: this is the SDK generation fix to handle the new Core namespace

**Review by ssanjay1 — APPROVED** [review]
Approving to unblock, let's address comments if possible

File: packages/platform-sdk-generator/src/model/Model.ts (hunk 1 above)
Comment (bryantpark04) [reply packages/platform-sdk-generator/src/model/Model.ts]: Will add in the legacy client long aggregation fix PR

**Pull request merged**